documentation/docs/15-configuration.md

```diff
@@ -268,6 +269,8 @@ See [Prerendering](/docs/page-options#prerender). An object containing zero or m
     export default config;
     ```
 
+- `origin` — the value of `url.origin` during prerendering
```

Comment:
is there a usecase we should include here? without having studied the issue, it's not clear to me why/when you'd use this option

Reply:
you might need it for the purposes of constructing an RSS feed or things like that. there's an example here: https://github.com/sveltejs/kit/issues/5444#issuecomment-1182612954

though i'm not sure how to communicate that in the description here without going into way too much detail or just being circular ('the value of `url.origin`, for when you need `url.origin` to have a specific value')

it could be done with an environment variable i suppose, but this feels more semantic, and means that we get rid of the weird `http://sveltekit-prerender` magic string in favour of configuration